server/apps/immich/src/modules/background-task/background-task.processor.ts

```diff
@@ -29,11 +28,14 @@ export class BackgroundTaskProcessor {
         }
       });
 
-      fs.unlink(asset.resizePath, (err) => {
-        if (err) {
-          console.log('error deleting ', asset.originalPath);
-        }
-      });
+      // TODO: what if there is no asset.resizePath. Should fail the Job?
```

Comment:
Before I put the `if`, when `asset.resizePath` was undefined it would probably throw an error. Since I made it safe adding the `if`, it would not longer  throw an error.

I imagine the Job might fail if there is an unhandled error thrown 🤔. An error from the callback of `fs.unlink` doesn't seem like it would make the Job fail either. Maybe it should use something like `job.moveToFailed()`, or maybe just ignoring it it's fine in this case 😄